packages/sync-actions/test/product-sync-prices.spec.js

```diff
@@ -567,7 +730,7 @@ describe('Actions', () => {
               fractionDigits: 2,
             },
             id: '7960b455-ab04-4722-a8b2-431460d60012',
-            custom: {
+            customField: {
```

Comment:
This looks wrong, please revert this change.

Reply:
done